### DISCUSSION TITLE
Potential design for making the delete action harder

### DISCUSSION THREAD

**Discussion by frankfzhao** [twilio-labs/paste#1486] [Q&A] [answered]
SendGrid Macerating Campaign has a special use case that currently a lot of customers experiencing the issue that they accidentally delete all their contacts by 2 clicks.
This design iteration aims to make it harder for users to perform deleting all contacts.
My thought is to add a text field and ask users to type in "Delete all contacts" to enable the actual destructive button which allow users to perform the delete action.
![Delete Contacts](https://user-images.githubusercontent.com/82524550/119418872-3f7a9500-bcb6-11eb-8ba8-f34e9be4267c.png)
![All contacts-2](https://user-images.githubusercontent.com/82524550/119418915-5620ec00-bcb6-11eb-9bf8-003c725f47d6.png)
Any suggestion on the copy / usage on this new workflow?

Thanks!🙏

**Comment by TheSisb** [comment, marked as answer]
Hi @frankfzhao,

This is a good suggestion and one we recommend for high severity actions on our [delete pattern documentation](https://paste.twilio.design/patterns/delete/#high-severity).

For copy, please be mindful of capitalization like in your header. Here's our specific [guidelines on capitalization](https://paste.twilio.design/content/product-style-guide#capitalization). If you have further questions about copy we suggest you reach out to @ctecks.

Let me know if you have any other questions :)

**Reply by frankfzhao** [reply]
Thanks! We will implement this under SendGrid UI which has some differences comparing to Paste for the capitalization. I will keep you posted when we move to Paste
